linera-views/src/backends/scylla_db.rs

```diff
@@ -444,6 +509,24 @@ impl ScyllaDbClient {
     }
 }
 
+async fn build_session(uri: &str) -> Result<Session, ScyllaDbStoreInternalError> {
+    let policy = DefaultPolicy::builder().token_aware(true).build();
+    let default_profile = ExecutionProfile::builder()
+        .load_balancing_policy(policy)
+        .retry_policy(Arc::new(DefaultRetryPolicy::new()))
+        .consistency(Consistency::LocalQuorum)
+        .build();
+    let handle = default_profile.into_handle();
+    SessionBuilder::new()
+        .known_node(uri)
+        .default_execution_profile_handle(handle)
+        .compression(Some(Compression::Lz4))
+        .build()
+        .boxed()
+        .await
+        .map_err(Into::into)
+}
```

Comment:
There are a number of choices being made here, and so it would be nice to have some documentation.